hoot-services/pom.xml

```diff
@@ -36,7 +36,7 @@
         <javax.servlet-api.version>3.0.1</javax.servlet-api.version>
         <jersey.version>2.30.1</jersey.version>
         <json-simple.version>1.1.1</json-simple.version>
-        <log4j2.version>2.16.0</log4j2.version>
+        <log4j2.version>2.17.0</log4j2.version>
```

Comment:
@brianhatchl: It looks like there's another new version release to fix another vulnerability: `2.17.1`